### PR TITLE
Fix: support ocm managedcluster in CLI

### PR DESCRIPTION
**Pull request by Somefive**
Signed-off-by: Somefive <yd219913@alibaba-inc.com>

### Description of your changes

1. For velad (backended by k3s), the ConfigMap "cluster-info" in "kube-public" does not exist. To let OCM cluster joinable in velad, the CLI needs to create the bootstrap cluster-info configmap if not exists.
2. Upgrade vela-prism to distinguish OCM managed cluster secret and native cluster secret.

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

Related Docs: https://github.com/kubevela/kubevela.io/pull/733

### Special notes for your reviewer

- `vela cluster join`, `vela cluster list`, `vela cluster probe` supports OCM ManagedCluster after this PR.
- `vela cluster detach` for OCM managed clusters will encounter troubles which needs manual operations for now.
- `vela cluster alias` and `vela cluster labels` does not support OCM ManagedCluster completely for now.

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]
Shall we backport to 1.4？

**Review by yangsoon — APPROVED** [review]

File: pkg/multicluster/cluster_management.go [hunk 1]
```diff
@@ -84,17 +84,25 @@ func (clusterConfig *KubeClusterConfig) Validate() error {
 	return nil
 }
 
-// RegisterByVelaSecret create cluster secrets for KubeVela to use
-func (clusterConfig *KubeClusterConfig) RegisterByVelaSecret(ctx context.Context, cli client.Client) error {
-	if err := ensureClusterNotExists(ctx, cli, clusterConfig.ClusterName); err != nil {
-		return errors.Wrapf(err, "cannot use cluster name %s", clusterConfig.ClusterName)
+// PostRegistration try to create namespace after cluster registered. If failed, cluster will be unregistered.
+func (clusterConfig *KubeClusterConfig) PostRegistration(ctx context.Context, cli client.Client) error {
+	if clusterConfig.CreateNamespace != "" {
```
Comment (zzxwill) [inline pkg/multicluster/cluster_management.go:89]: To keep the body simple, you can check whether CreateNamespace is empty and return nil  :)

**Review by zzxwill — APPROVED** [review]
/lgtm

File: pkg/multicluster/cluster_management.go (hunk 1 above)
Comment (Somefive) [reply pkg/multicluster/cluster_management.go:89]: Good idea!

**Review by FogDong — APPROVED** [review]

**Pull request merged**